apps/test/unit/code-studio/searchAssetsTest.js

```diff
@@ -0,0 +1,76 @@
+/** @file Test of searchAssetsTest.js. */
+
+var assert = require('assert');
+import {searchAssets} from '@cdo/apps/code-studio/assets/searchAssets';
+var animationLibrary = require('@cdo/apps/gamelab/animationLibrary.json');
+var soundLibrary = require('@cdo/apps/code-studio/soundLibrary.json');
```

Comment:
can we use import instead of require? if no, we should still use const instead of var